lib/tlGL/RenderVideo.cpp

```diff
@@ -131,7 +131,7 @@ namespace tl
                         videoData[1],
                         math::BBox2i(0, 0, p.size.w, p.size.h),
                         imageOptions.size() > 1 ? std::make_shared<timeline::ImageOptions>(imageOptions[1]) : nullptr,
-                        !displayOptions.empty() ? displayOptions[1] : timeline::DisplayOptions());
+                        displayOptions.size() > 1 ? displayOptions[1] : timeline::DisplayOptions());
```

Comment:
I don't think this change is supposed to be part of this PR?

Reply:
This was a previous PR which I thought you merged into main.   See:

Fixed an erroneous comparison on displayOptions array. #54

commit 51e0ae51200e02e9d683a55e41172f5d2fa8b33b

Reply:
I did merge it into main, is your branch up to date?